src/scss/custom/profile/_profileCustomClasses.scss

```diff
@@ -21,3 +21,9 @@ Version: 1.0.0
   max-width: 300px;
   max-height: 300px;
 }
+#skillsList {
+  li {
+    width: fit-content !important;
```

Comment:
Is !important really necessary? it shouldn't be used unless you absolutely have to.

Reply:
I can check that

Reply:
it was not necessary, I removed it now